TAC/Meetings/2022/2022-04-07/CCC_TAC_Minutes-2022-04-07.md

```diff
@@ -41,107 +41,92 @@
 
 ### Roll Call / Attendance
 
 #### Voting members
 
-* Dan Middleton (Intel)
+<!-- * Dan Middleton (Intel)
 * Dave Thaler (Microsoft)
+* Eric Northup (Meta)
 * Lily Sturman (Red Hat)
-* Giuseppe Giordano (Accenture)
-* Tate, Hongliang Tian (Ant Group)
-* Thomas Fossati (ARM)
+* Zhipeng (Howard) Huang (Huawei)
+* Thomas Fossati (Arm) -->
 
-#### Attendees
+#### Other attendees
 
+<!--* Ben Fischer (Red Hat)
+* Brian Warner (Linux Foundation)
 * Eric Voit (Cisco)
+* Helen Lau (Linux Foundation)
+* Henk Birkholz ()
+* Kurt Taylor (Linux Foundation)
+* Naveen Cherukuri (Nvidia)
+* Mike Bursell (Profian)
 * Nick Vidal (Profian)
 * Penglin Yang (China Mobile)
-* Radhika Jandhyala (Microsoft)
-* Ravi Sahita (Rivos)
-* Samuel Ortiz (Apple)
-* Shalom Shefa (Cisco)
+* Sneha Damle (R3)
 * Simon Johnson (Intel)
 * Steve Van Lare (Anjuna)
+* Xinxin Fan (IoTeX) -->
 
 ### Meeting minute approvals
 
 #### Approved by voting members
 
-<!-- **RESOLVED:** That the minutes of the [January 13, 2022](../2022-01-13/TAC_Minutes-2022-01-13.pdf) meeting of the Technical Advisory Council meeting of the Confidential Computing Consortium as distributed to the members of the TAC in advance of this meeting are hereby adopted and approved. -->
+<!--
+**RESOLVED:** That the minutes of the [March 10, 2022](../2022-03-10/CCC_TAC_Minutes-2022-03-10.md) meeting of the Technical Advisory Council meeting of the Confidential Computing Consortium as distributed to the members of the TAC in advance of this meeting are hereby adopted and approved.-->
 
 #### Pending approval
 
-* [December 16, 2021](../../2021/2021-12-16/TAC_Minutes-2021-12-16.pdf), further updates needed
-* [February 10, 2022](https://github.com/confidential-computing/governance/pull/93/files)
-* [February 24, 2022](https://github.com/confidential-computing/governance/pull/96/files)
+* [March 10, 2022](../2022-03-10/CCC_TAC_Minutes-2022-03-10.md)
 
 ### Open Action Items
 
-1. **[Mike, Eric, Stephen, Zongmin]** Recommend diversity and inclusion training to their projects and report back to the TAC on whether the maintainers or the contributors will be taking it and when the expected completion date is.
+1. **[Stephen, Zongmin]** Recommend diversity and inclusion training to their projects and report back to the TAC on whether the maintainers or the contributors will be taking it and when the expected completion date is.
 
 1. **[Stephen Walli, Dan Middleton, Aeva Black]** Develop a CoC escalation process, to be presented to TAC and governing board.
-    * Dan Middleton has opened a PR for review
+    * Dan Middleton has [opened a PR for review](https://github.com/confidential-computing/governance/pull/92)
 
 1. **[Mike, Eric V, Stephen, Zongmin]** Recommend diversity and inclusion training to their projects and report back to the TAC on whether the maintainers or the contributors will be taking it and when the expected completion date is.
```

Comment:
Duplicate of line 87, remove.

Reply:
Thanks @dthaler and @dcmiddle, I've added a note to the March 10 minutes that this will be tracked via PR instead. I moved it to completed actions, and removed it from the April 7 draft. Commit coming shortly.